test/controllers/settings/company/general_profiles_controller_test.rb

```diff
@@ -4,11 +4,31 @@
 
 class Settings::Company::GeneralProfilesControllerTest < ActionDispatch::IntegrationTest
   test "should open general company settings" do
-    skip "TODO: Functionality in the process of implementation."
-    sign_in accounts(:interviewer_account)
+    sign_in accounts(:admin_account)
 
     get settings_company_general_path
 
     assert_response :success
   end
+
+  test "should update company name if name is valid" do
```

Comment:
would be good to check that before updating company name is not "New Name" and after updating is "New Name"